backend/hitas/models/housing_company.py

```diff
@@ -99,3 +99,21 @@ class Meta:
 
     def __str__(self):
         return self.display_name
+
+
+class HousingCompanyMarketPriceImprovement(HitasModel):
```

Comment:
Since there are four `Improvement` models, i could have made sense to have an abstract parent model these inherit from

Reply:
I originally did that, but the foreign key caused some pain (2 of them are joined to housing company and 2 to apartments + `related_name` causes some nasty names). Back then these models were a bit different but now I agree the three common fields could move moved to an abstract base class

Reply:
I added one for commit to address this